internal/plans/objchange/objchange_test.go

```diff
@@ -2036,14 +2036,14 @@ func TestProposedNew(t *testing.T) {
 				)),
 			}),
 			cty.ObjectVal(map[string]cty.Value{
-				"list_obj": cty.ListVal([]cty.Value{
-					cty.ObjectVal(map[string]cty.Value{
-						"obj": cty.ObjectVal(map[string]cty.Value{
-							"optional": cty.StringVal("prior"),
-							"computed": cty.StringVal("prior computed"),
+				"list_obj": cty.NullVal(cty.List(
```

Comment:
Should the comment on ll. 1992-1993 be removed?
"The entire prior nested value should be represented in the proposed new object if the configuration is null."

Reply:
ooh, that's a good catch! The result is good, let me check that it's what we actually want at this point in the stack of PRs though (this was broken up from a single huge PR).

Reply:
ok, this is the test I thought its was! updated the comment to describe the more accurate planning result.